### PR TITLE
LBSD-895 All editors in the blog team should be able to edit blog set…

### PR DESCRIPTION
**Pull request by vladnicoara**
…tings

### CODE REVIEW

File: client/app/scripts/liveblog-security.service.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ angular.module('liveblog.security', [])
             return $rootScope.currentUser._id === archive.original_creator || canPublishAPost();
         }
         function canAccessSettings(archive) {
-            return canCreateABlog() && isUserOwnerOrAdmin(archive);
+            return canCreateABlog() && (isUserOwnerOrAdmin(archive) || isMemberOfBlog(archive));
```
Comment (ancafarcas) [inline client/app/scripts/liveblog-security.service.js]: after the change you made here, all the members of the blogs will have the rights to edit the settings, not only the editors, right?

Comment (nistormihai) [inline client/app/scripts/liveblog-security.service.js]: Anca is right it should be only for '"editor" role that are in the blog team'
this grands right for all the users not only for "editor"s.

Comment (vladnicoara) [inline client/app/scripts/liveblog-security.service.js]: it must have canCreateABlog right as well and that means admins and editors. Contributors do not have the rights to create blog. Tested the thing. You can check how it works on the branch

Comment (ancafarcas) [inline client/app/scripts/liveblog-security.service.js]: You're right! Despite of what is understandable from the code, if you log in with contributor account you don't have the rights to create blogs. Tested on the branch.
So :+1:

**Pull request merged**